recipes/fcl/all/conanfile.py

```diff
@@ -47,10 +47,13 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("eigen/3.4.0")
-        self.requires("libccd/2.1")
+        # Used in fcl/common/types.h public header
+        self.requires("eigen/3.4.0", transitive_headers=True)
+        # Used in fcl/narrowphase/detail/convexity_based_algorithm/support.h
+        self.requires("libccd/2.1", transitive_headers=True, transitive_libs=True)
```

Comment:
As far as I can see - this still does not require transitive_libs to be unconditionally set to true, has this been verified?